.github/workflows/docker-build-push-railgun.yaml

```diff
@@ -0,0 +1,79 @@
+name: Build and push Railgun images
```

Comment:
For consistency with #1410, IMO this file should have the `2x-` prefix and the `name` should be something like `2.x Build and Push`